### ISSUE TITLE
ERCF_HOME  Must be heated

### ISSUE THREAD

**Issue by VeryColdly** [moggieuk/ERCF-Software-V3#49]
![image](https://user-images.githubusercontent.com/99728701/235950933-b2ef228c-5651-4105-81da-7c6f024a54c0.png)

**Comment by favoritewky** [comment]
> ![image](https://user-images.githubusercontent.com/99728701/235950933-b2ef228c-5651-4105-81da-7c6f024a54c0.png)

Yes,and you can config the `min_temp_extruder: 180	` in `ercf_parameters.cfg`

**Comment by moggieuk** [comment]
If calibrate single is changing then there must be an issue with encoder or friction causing slippage.

**Comment by moggieuk** [comment]
The issue with ERCF_LOAD followed by ERCF_HOME has been fixed.
